blocks/hero/hero.css

```diff
@@ -12,28 +12,85 @@ main .hero {
   position: relative;
   padding: 32px;
   min-height: 300px;
+  display: flex;
+  max-width: var(--max-content-width);
+  margin: auto;
+}
+
+main .hero .left-box {
+  background: #024c99;
+  min-height: 400px;
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  overflow: hidden;
+  position: relative;
+}
+
+main .hero .right-box {
+  flex: 1;
+  display: none;
+}
+
+main .hero .left-box .bg-circle-1 {
```

Comment:
I'd suggest taking the common style from these and putting them in a new class `bg-circle` then just do the positioning with for 1 and 2 individually.

FWIW, I think if I was doing this I'd also probably just do this with pseudo-elements

Reply:
I tried to do with `before` and `after` but I might be missing something as I'm getting this error from the browser:

> The position: static property prevents left/bottom from having an effect. 

```
main .hero .hero-content-container:before {
  background: var(--clr-light-blue); 
  display: block;
  position: absolute;
  opacity: .4;
  width: 390px;
  height: 390px;
  right: -90px;
  top: -210px;
  border-radius: 195px;
}

main .hero .hero-content-container:after {
  background: var(--clr-light-blue); 
  display: block;
  position: absolute;
  opacity: .4;
  width: 220px;
  height: 220px;
  left: -110px;
  bottom: 40px;
  border-radius: 110px;
}
```